editor/script_editor_debugger.cpp

```diff
@@ -645,7 +645,11 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 							Ref<Script> script(var);
 							if (!script.is_null()) {
 								ScriptInstance *script_instance = script->placeholder_instance_create(debugObj);
-								debugObj->set_script_and_instance(var, script_instance);
+
+                                // fix: VisualScript resource
```

Comment:
Rename it to "Fix: VisualScript resource."

Reply:
Style-wise that's correct, but content-wise I'm not sure this comment is useful. What does it fix exactly?